.github/workflows/test.yml

```diff
@@ -20,9 +20,8 @@ jobs:
           MYSQL_ALLOW_EMPTY_PASSWORD: true
       postgres:
         image: postgres:13.3
-        restart: always
         ports:
-          - "5432:5432"
+          - 5432:5432
         env:
           - POSTGRES_USER: root
           - POSTGRES_PASSWORD: pass
```

Comment:
does postgres have a way to use default root user name and default password?

Reply:
Actually it was throwing error without these, and other option was to allow all host connections to use the db without authentication ```POSTGRES_HOST_AUTH_METHOD=trust```, but I think this is not recommended.

Reply:
Btw, I have used this as a root to create another temporary user for that particular db. So, we are not accessing the extractor as a root, and I think it should be fine.